src/store/modules/vote/index.js

```diff
@@ -0,0 +1,50 @@
+import { call, put, takeLatest } from 'redux-saga/effects';
+
+import { VOTE_ON_ARTICLE, VOTE_SUCCESSFULL, ARTICLE_VOTE_COUNT } from 'modules/vote/types';
```

Comment:
Please let's change the action types to the below i.e.

VERB_NOUN_ACTION i.e. VOTE_ARTICLE_REQUEST, VOTE_ARTICLE_FAILURE, VOTE_ARTICLE_SUCCESS etc

```suggestion
import { VOTE_ARTICLE_REQUEST, VOTE_ARTICLE_SUCCESS, GET_VOTE_COUNT_REQUEST } from 'modules/vote/types';
```

Reply:
Please correct throughout the file